config/300-resolutionrequest.yaml

```diff
@@ -33,8 +33,8 @@ spec:
       - resolutionrequests
   versions:
     - name: v1alpha1
-      served: true
-      storage: true
+      served: false
```

Comment:
would it make sense to keep served: true and add deprecated: true? Is it possible that people have resolutionrequests saved in their cluster and this could break them?

Reply:
When you say they are short-lived, do you mean they're not referred to again after resolution, or they are deleted after resolution? If the latter, I think we still need to serve v1alpha1 because there will still be resolution requests present on some clusters. But ofc that doesn't matter post change-- thanks!

Reply:
Not referred to again - we probably should be cleaning them up automatically at some point, though. But yeah, thanks for bringing this up!

Reply:
@abayer aren't they "owned" (through ownerRef) by the `PipelineRun` that created it ? If yes, then yes they should be deleted when the `PipelineRun` is deleted (same applies for `TaskRun`).

Reply:
They are, I tested it earlier, and they're deleted together with the `TaskRun` / `PipelineRun`.

Reply:
@abayer Since this is likely to be in v0.41 which is still 10+ days away, I think it's still worth sending a notice on the mailing list, and we should have this mentioned in the release notes as well, so that if anyone implemented they're own resolver they'll know they must updated it before switching to v0.41.

Reply:
👍 I'll send that as soon this merges.